core/lib/basic_types/src/prover_dal.rs

```diff
@@ -229,3 +230,33 @@ impl FromStr for GpuProverInstanceStatus {
         }
     }
 }
+
+#[derive(Debug, EnumString, Display)]
+pub enum ProofCompressionJobStatus {
```

Comment:
Isn't there a similar enum in types? May we reuse it here?

Reply:
Yes! This is the one that you are talking about, I just moved it to the right module.